pkg/mock/fileUtils.go

```diff
@@ -21,8 +21,8 @@ type fileProperties struct {
 //FilesMock implements the functions from piperutils.Files with an in-memory file system.
 type FilesMock struct {
 	files        map[string]*fileProperties
-	writtenFiles map[string]*[]byte
-	removedFiles map[string]*[]byte
+	writtenFiles []string
```

Comment:
Other possible approach would be to keep a map with some irrelevant dummy as value (e.g. always registered an empty byte slice). with that there would be no need for the `contains` function.